packages/shader-lab/src/preprocessor/PpParser.ts

```
@@ -39,6 +46,9 @@
     this.addPredefinedMacro("GL_ES");
     this._includeMap = includeMap;
     this._basePathForIncludeKey = basePathForIncludeKey;
+    // #if _VERBOSE
+    this._errors.length = 0;
```

Biome

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)